js/src/ui/Form/Input/input.js

```diff
@@ -128,75 +118,48 @@ export default class Input extends Component {
 
   render () {
     const { value } = this.state;
-    const { autoFocus, children, className, defaultValue, hideUnderline, disabled, error } = this.props;
-    const { focused, label, hint, onClick, multiLine, rows, type, min, max, step, style, tabIndex } = this.props;
-
-    const readOnly = this.props.readOnly || disabled;
-
-    const inputStyle = { overflow: 'hidden' };
-    const textFieldStyle = {};
-
-    if (readOnly) {
-      inputStyle.cursor = 'text';
-    }
-
-    if (hideUnderline && !hint) {
-      textFieldStyle.height = 'initial';
-    }
-
-    const underlineStyle = readOnly ? UNDERLINE_READONLY : UNDERLINE_NORMAL;
-    const underlineFocusStyle = focused
-      ? UNDERLINE_FOCUSED
-      : readOnly && typeof focused !== 'boolean' ? { display: 'none' } : null;
-
-    const textValue = typeof value !== 'string' && (value && value.props)
-      ? this.context.intl.formatMessage(
-          value.props,
-          value.props.values || {}
-        )
-      : value;
+    const {
+      children,
+      className,
+      defaultValue,
+      disabled,
+      error,
+      fluid,
+      focused,
+      label,
+      onClick,
+      style,
+      tabIndex,
+      type
+    } = this.props;
 
     return (
       <div className={ styles.container } style={ style }>
         { this.renderCopyButton() }
-        <TextField
-          autoComplete='off'
-          autoFocus={ autoFocus }
+        <InputUI
           className={ className }
-          defaultValue={ defaultValue }
-          errorText={ error }
-          floatingLabelFixed
-          floatingLabelText={ label }
-          fullWidth
-          hintText={ hint }
-          id={ NAME_ID }
-          inputStyle={ inputStyle }
-          max={ max }
-          min={ min }
-          multiLine={ multiLine }
-          name={ NAME_ID }
+          disabled={ disabled }
+          error={ error }
+          fluid={ fluid }
+          focus={ focused }
+          input={ value }
+          label={ label }
           onBlur={ this.onBlur }
           onChange={ this.onChange }
           onClick={ onClick }
           onKeyDown={ this.onKeyDown }
           onKeyUp={ this.onKeyUp }
           onFocus={ this.onFocus }
           onPaste={ this.onPaste }
-          readOnly={ readOnly }
-          ref='input'
-          rows={ rows }
-          step={ step }
-          style={ textFieldStyle }
+          placeholder={ defaultValue }
```

Comment:
placeholder === hint in the original version? (We use them everywhere as `hint`)